<launch>

  <!-- ROS Joystick driver -->
  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="autorepeat_rate" value="10" />
    <remap from="joy" to="joystick"/>
  </node>


  <!-- Logitech F710 Gamepad Mapper -->
  <node pkg="rr_control_input_manager" type="logitech_mapper.py" name="logitech_mapper_node" output="screen">
    <param name="mode" value="x" />
    <param name="joy_topic" value="joystick" />
    <param name="max_vel_drive" value="1.3" />
    <param name="max_vel_turn" value="4.5" />
  </node>


  <!-- Control input manager -->
  <node pkg="rr_control_input_manager" type="control_input_manager.py" name="control_input_manager_node">
  </node>

</launch>
